catalogData/elasticsearch24/3x-ha/k8s/statefulset-master.json

```diff
@@ -141,12 +142,34 @@
                 "value":"TAP"
               }
             ],
-            "imagePullPolicy": "Always"
+            "volumeMounts": [
+              {
+                "name": "data",
+                "mountPath": "/usr/share/elasticsearch/data"
+              }
+            ],
+            "imagePullPolicy": "IfNotPresent"
           }
         ],
         "restartPolicy": "Always",
         "dnsPolicy": "ClusterFirst"
       }
-    }
+    },
+    "volumeClaimTemplates": [
+      {
+        "metadata": {
+          "name": "data"
+        },
+        "spec": {
+          "accessModes": ["ReadWriteOnce"],
+          "storageClassName": "$storage_class",
+          "resources": {
+            "requests": {
+              "storage": "10Gi"
```

Comment:
Certainly no more than our production logsearch cluster which is using a grand total of 97MB.  I updated this to use the minimum EBS size of 1GB